repos/spack_repo/builtin/packages/py_numpy/package.py

```diff
@@ -223,6 +223,11 @@ def patch(self):
     conflicts("%oneapi", when="@1.23.0:1.23.3")
     # https://github.com/spack/spack/pull/44735
     conflicts("%oneapi", when="@2:")
+    #
+    # yoder:
+    conflicts(
+        "@:1.26.3", when="^python@3.12", msg="Known bug with @1.26.2, 1.26.3 when ^python@3.12"
```

Comment:
This is only a more complicated way of modifying the lower bound of this line, right?
```
depends_on("python@3.9:3.12", when="@1.26:2.0")
```
That doesn't look right to me, 3.12 is supported in the 1.26.x releases - the problem has to be somewhere else or specific to your setup.